modules/structs/repo.go

```diff
@@ -79,6 +79,7 @@ type Repository struct {
 	Created time.Time `json:"created_at"`
 	// swagger:strfmt date-time
 	Updated                       time.Time        `json:"updated_at"`
+	ArchivedAt                    *time.Time       `json:"archived_at"`
```

Comment:
The Migration is already done by setting the default to 0.

Reply:
No, this is also what I complained about:With your way, the column won't be created in the database at all, it will simply be read from at runtime.

Reply:
column will be added also without migration but it can create problems in future migrations if they will use this column.

https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md#design-guideline

Reply:
So I should run
```sql
ALTER TABLE repository
ADD COLUMN archived_unix INTEGER DEFAULT 0;
```
during migration?

Reply:
Column should be added using Sync2 but data update should be made using single update by setting new column to updated_unix column value

Reply:
Can you link to example Migration using this?

Reply:
https://github.com/go-gitea/gitea/pull/22664/files#r1171375981